core/dbt/parser/unit_tests.py

```diff
@@ -178,18 +189,21 @@ def _get_original_input_node(self, input: str, tested_node: ModelNode):
                 raise InvalidUnitTestGivenInput(input=input)
 
             if statically_parsed["refs"]:
-                for ref in statically_parsed["refs"]:
-                    name = ref.get("name")
-                    package = ref.get("package")
-                    version = ref.get("version")
-                    # TODO: disabled lookup, versioned lookup, public models
-                    original_input_node = self.manifest.ref_lookup.find(
-                        name, package, version, self.manifest
-                    )
+                ref = list(statically_parsed["refs"])[0]
+                name = ref.get("name")
+                package = ref.get("package")
+                version = ref.get("version")
+                # TODO: disabled lookup, versioned lookup, public models
+                original_input_node = self.manifest.ref_lookup.find(
+                    name, package, version, self.manifest
+                )
             elif statically_parsed["sources"]:
-                input_package_name, input_source_name = statically_parsed["sources"][0]
+                source = list(statically_parsed["sources"])[0]
+                input_source_name, input_name = source
                 original_input_node = self.manifest.source_lookup.find(
-                    input_source_name, input_package_name, self.manifest
+                    f"{input_source_name}.{input_name}",
+                    self.root_project.project_name,
```

Comment:
I believe it should be possible to write a unit test where a source input is coming from an imported package (cc @graciegoheen to confirm)

Does this setting restrict that capability? I think it should be `None` to find a source in any package

Reply:
Yes, I think we should make it possible to add a unit test to a model that depends on a source defined in a package.

Reply:
Will change package to None.

Reply:
Done.